app/components/assets/assets-index/advanced-asset-columns.tsx

```diff
@@ -67,7 +67,7 @@ export function AdvancedIndexColumn({
 
   if (isCustomField) {
     const fieldName = column.replace("cf_", "");
-    const field = item.customFields.find(
+    const field = item.customFields?.find(
```

Comment:
I have fixed the issue that caused this so its not really needed, but its also not bad to have this extra check.